docs/csharp/tuples.md

```diff
@@ -25,10 +25,20 @@ In this topic, you'll learn the language rules governing Tuples in C# 7,
 different ways to use them, and initial guidance on working with Tuples.
 
 > [!NOTE]
-> The new tuples features require the `System.ValueTuple` type. For Visual Studio 2017,
-> you must add the NuGet package [System.ValueTuple](https://www.nuget.org/packages/System.ValueTuple/), available on the NuGet Gallery.
-> Without this package you may get a compilation error similar to `error CS8179: Predefined type 'System.ValueTuple``2' is not defined or imported`
```

Comment:
We should add CS8137 and CS8179 to the sorry-we-don-t-have-specifics... file in the C# compiler error messages directory. I'll open a quick PR to do it and add a few others.